CHANGES.md

```diff
@@ -6,7 +6,7 @@ Change Log
 * Breaking changes
   *
 * Deprecated
-  *
+  * `Globe.depthTestAgainstTerrain` was deprecated.  It will be removed in 1.14.  Use `Scene.depthTestAgainstTerrain` instead.
```

Comment:
What do you think of just leaving `Globe.depthTestAgainstTerrain` because:
* This breaking change will definitely break some apps
* Once we have terrain on by default and vector data on terrain, won't we just remove it?